### PR TITLE
API7 Update & /needonly command

### PR DESCRIPTION
**Pull request by xStarryNight**
updates the code to Net6.0 and bumps the version to API7
+ Adds a command to /needonly to need for everything you can, and pass for what you can't.

### CODE REVIEW

**Comment by Iaotle** [comment]
Hey, your /needonly command has a typo in it. Please fix, and I can merge :D

**Comment by xStarryNight** [comment]
where at? I don't see any typos

**Comment by xStarryNight** [comment]
oh I see it, I'll adjust it then. I've added something in the new code as well to pass on everything that hasn't been rolled on for /need and /greed as well (for when you already have an item)

**Comment by Iaotle** [comment]
Cheers mate! If you could build it and send it over I can upload it to my dalamud repo as well.

File: Plugin.cs [hunk 1]
```diff
@@ -77,11 +78,16 @@ public void NeedCommand(string command, string args)
...
                         ++num2;
                     }
+                    else
+                    {
+                        RollItem(RollOption.Pass, index);
+                        ++num3;
+                    }
                 }
             }
             if (!config.EnableChatLogMessage)
```
Comment (imgidedin) [inline Plugin.cs:93]: Hey, it is not related to your commit itself, but is this line being stored when we check/uncheck the checkbox in the config? I don't think it was. Could you look at it? Thanks for the PR, you rockl!

Comment (xStarryNight) [reply Plugin.cs:93]: I did look into the config not saving the checkbox selection and it seems like the code doesn't save any changes to it. I'll make a pr when I'm able to fix it.

**Pull request merged**